test/functional/feature_datacarrier.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020-2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test datacarrier functionality"""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.script import (
+    CScript,
+    OP_RETURN,
+)
+from test_framework.wallet import (
+    MiniWallet
+)
+from test_framework.messages import (
+    CTxOut,
+)
+from test_framework.util import (
+    assert_raises_rpc_error,
+)
```

Comment:
Done in https://github.com/bitcoin/bitcoin/commit/10bfe1204fed71ff10d6bbac90912804894d454b.

Reply:
AFAIK, we usually prefer multi-line over single-line imports in order to avoid potential merge conflicts. I opened a PR to hopefully get some consensus on how strict this (so far unwritten) rule should be followed: https://github.com/bitcoin/bitcoin/pull/25811